File: Serene/Serene.Script/Meeting/MeetingAgendaRelevant/MeetingAgendaRelevantGrid.cs

namespace Serene.Meeting
{
    using jQueryApi;
    using Serenity;
    using System;
    using System.Collections.Generic;
    using System.Runtime.CompilerServices;

    [ColumnsKey("Meeting.MeetingAgendaRelevant"), IdProperty("MeetingAgendaRelevantId"), IsActiveProperty("IsActive")]
    [DialogType(typeof(MeetingAgendaRelevantDialog)), LocalTextPrefix("Meeting.MeetingAgendaRelevant"), Service("Meeting/MeetingAgendaRelevant")]
    public class MeetingAgendaRelevantGrid : EntityGrid<MeetingAgendaRelevantRow>
    {
        public MeetingAgendaRelevantGrid(jQueryObject container)
            : base(container)
        {
        }
    }

    // Please remove this partial class or the first line below, after you run ScriptContexts.tt
    [Imported, Serializable, PreserveMemberCase] 
    public partial class MeetingAgendaRelevantRow
    {
    }
}

File: Serene/Serene.Web/Modules/Meeting/MeetingAgendaRelevant/MeetingAgendaRelevantGrid.ts

namespace Serene.Meeting {
    
    @Serenity.Decorators.registerClass()
    export class MeetingAgendaRelevantGrid extends Serenity.EntityGrid<MeetingAgendaRelevantRow, any> {
        protected getColumnsKey() { return 'Meeting.MeetingAgendaRelevant'; }
        protected getDialogType() { return MeetingAgendaRelevantDialog; }
        protected getIdProperty() { return MeetingAgendaRelevantRow.idProperty; }
        protected getLocalTextPrefix() { return MeetingAgendaRelevantRow.localTextPrefix; }
        protected getService() { return MeetingAgendaRelevantService.baseUrl; }

        constructor(container: JQuery) {
            super(container);
        }
    }
}